.vscode/settings.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
seems like this was commited before `vscode` was put in `.gitignore`

Reply:
thnx, have removed this now